src/s2wasm.h

```diff
@@ -1167,6 +1173,42 @@ class S2WasmBuilder {
         }
       }
     }
+    if (startFunction.size()) {
```

Comment:
This is checking whether a `--start` was provided: an empty string means no. It's now `.is()` because of `Name`'s API.